esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -2135,3 +2135,13 @@ StringArray & getRoxieDirectAccessPlanes(StringArray & planes, StringBuffer &def
         planes.append(defaultPlane);
     return planes;
 }
+
+extern TPWRAPPER_API SecAccessFlags getDropZoneScopePermissions(IEspContext & context, const char * dropZoneName, const char * dropZoneServer, const char * dropZonePath)
```

Comment:
see commet in TpContainer.cpp re. removing unusued param

Reply:
Is this related to: https://github.com/hpcc-systems/HPCC-Platform/pull/16710#discussion_r1029416424 ?

Reply:
Yes